frontend/public/json-example/admin_transactionGdt_list.json

```diff
@@ -0,0 +1,45 @@
+{"state":"success", "gdt": 
```

Comment:
Is this file still needed? If not, please remove.

Reply:
removed